deluge/tests/test_files_tab.py

```diff
@@ -96,6 +97,8 @@ def test_files_tab(self):
         self.assertTrue(ret)
 
     def test_files_tab2(self):
+        if windows_check():
+            raise unittest.SkipTest('on windows \ != / for path names')
```

Comment:
A possible issue with `files_list`!

However, inside Python Windows should handle this so perhaps the test needs tweaked.